das/state.go

```diff
@@ -146,8 +153,10 @@ func (s *coordinatorState) updateHead(newHead uint64) {
 		log.Infow("found first header, starting sampling")
 	}
 
+	oldHead := s.networkHead
 	s.networkHead = newHead
-	log.Debugw("updated head", "from_height", s.networkHead, "to_height", newHead)
+
+	log.Debugw("updated head", "from_height", oldHead, "to_height", newHead)
```

Comment:
Also looks like  keys for values in the log are a bit off. Do you mind to update them for something better while we are on it? Like "old_head" and "new_head" or similar.